pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -862,3 +862,21 @@ def test_read_seek(all_parsers):
             actual = parser.read_csv(file)
         expected = parser.read_csv(StringIO(content))
     tm.assert_frame_equal(actual, expected)
+
+
+@pytest.mark.xfail(
+    reason="the 'pyarrow' engine does not support separators with more than one "
+    r"character, except for '\\t'. Please use a supported separator "
+    "or a different engine."
+)
+def test_pyarrow_with_multi_character_separator(all_parsers):
+    # GH#52554
+    data = "a b c\n1 2 3\n4 5 6"
+    parser = all_parsers
+    msg = (
+        "the 'pyarrow' engine does not support separators with more than one "
+        r"character, except for '\\t'. Please use a supported separator "
+        "or a different engine."
+    )
+    with pytest.raises(ValueError, match=msg):
+        parser.read_csv(StringIO(data), engine="pyarrow", sep=r"\s+")
```

Comment:
I think the `all_parsers` feature is only useful if you want to test different engines in the same test. You can call just `pandas.read_csv` here, and remove the fixture (the `all_parsers` parameter)

Reply:
Another misunderstanding, i will fix this as well.